.github/workflows/license_check.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     name: Scan code for licenses
     steps:
     - name: Checkout the code
-      uses: actions/checkout@v1
+      uses: actions/checkout@v3
```

Comment:
Nice catch but this should be in a different commit please, not really a "typo".

Reply:
Reverted the change, will address this in a separate issue.